extensions/workloads/pods/verify.go

```diff
@@ -5,11 +5,11 @@ import (
 	"time"
 
 	provv1 "github.com/rancher/rancher/pkg/apis/provisioning.cattle.io/v1"
-	projectv3 "github.com/rancher/rancher/pkg/client/generated/project/v3"
```

Comment:
Whats the reason for swapping to appv1? Are these imports equivalent?

Reply:
No, one is generated for norman, and the other is the native struct we want for a v1 call. This should have never been merged like this and if it did, it should come from our own generated struct. There is so much wrong with with this import.

Reply:
Understood 😅